addons/binding/org.openhab.binding.alarm/README.md

```diff
@@ -0,0 +1,114 @@
+# Alarm Binding
+
+This is the binding for an alarm controller inspired by [Aritech](https://aritech-security.de) and [Abus](https://www.abus.com)  
+
+The binding is kept very simple and allows you to create alarm controllers with a configurable amount of alarm zones. Each alarm zone has a type like ACTIVE, SABOTAGE, EXIT_ENTRY, ... and you can bind a (Window/Motiondetector/...) Contact to each alarm zone. You can also send some commands to the controller for arming and disarming.
+
+## Supported Bridges
+
+None, this binding does not need a bridge
+
+## Supported Things
+
+This binding supports only one Thing: An alarm controller
+
+## Discovery
+
+On discovery, the binding creates a default alarm controller
+
+## Thing Configuration
+
+One alarm controller is created automatically, you can handle it in PaperUI.  
+
+You can also configure one or multiple controllers (things) manually:
+
+```java
+alarm:controller:home [alarmZones=10, entryTime=30, exitTime=30, passthroughTime=30, alarmDelay=30] {
+    Channels:
+        Type alarmZone : alarmZone_1 "My alarm zone"    [ type = "ACTIVE" ]
+        Type alarmZone : alarmZone_2 "My sabotage zone" [ type = "SABOTAGE" ]
+}
+```
+
+| Thing config | Description |
```

Comment:
Aligning all the columns in the tables with spaces would make the tables easier to read with plain text viewers. E.g. use an online tool like  https://www.tablesgenerator.com/markdown_tables to easily do this.